src/drt/src/pa/FlexPA.h

```diff
@@ -137,233 +138,235 @@ class FlexPA
   void prepPoint();
   void getViasFromMetalWidthMap(
       const Point& pt,
-      frLayerNum layerNum,
+      frLayerNum layer_num,
       const gtl::polygon_90_set_data<frCoord>& polyset,
-      std::vector<std::pair<int, frViaDef*>>& viaDefs);
+      std::vector<std::pair<int, frViaDef*>>& via_defs);
   template <typename T>
-  int prepPoint_pin(T* pin, frInstTerm* instTerm = nullptr);
+  int prepPointPin(T* pin, frInstTerm* inst_term = nullptr);
   template <typename T>
-  void prepPoint_pin_mergePinShapes(
-      std::vector<gtl::polygon_90_set_data<frCoord>>& pinShapes,
+  void prepPointPinMergePinShapes(
```

Comment:
I am not a big fan of having _ in the function names, but in the current state of the code, I think it's necessary for following the helper functions. For example, I can understand that prepPoint_pin_mergePinShapes is called by prepPoint_pin which is called by prepPoint.
later we can do better refactor to get rid of this _